### ISSUE TITLE
WIL botany Roxburgh / linnaeus

### ISSUE THREAD

**Issue by drdhaval2785** [sanskrit-lexicon/csl-orig#318]
Dear Jim,
The names such as `Agrostis linearis, Lin.`
and  `Panicus dactilon, Rox.`
In this, Lin / L / R / Rox etc are treated as integral part of plant name.
They are the scientists who gave this name. Most probably it will be these two scientists.

So tagging should be
`<bot>Agrostis linearis, Lin.</bot>` and `<bot>Panicus dactilon, Rox.</bot>`

Some scientist may be able to shed further light on this.

**Comment by gasyoun** [comment]
> They are the scientists who gave this name.

Yes, we have spoken about it in regards of MW, but nobody ever even planned to clean WIL.

**Comment by gasyoun** [comment]
@drdhaval2785 remember me.

**Comment by drdhaval2785** [comment]
I think we now have a tag 'abot' for abbreviation in botanical tags. We have used it in PWG. I guess we can use it in WIL too.

**Comment by drdhaval2785** [comment]
PWG has the following abots.
```
rg -o '<abot>[^<]+[.]</abot>' v02/pwg/pwg.txt | sort | uniq 
<abot>Buch.</abot>
<abot>DC.</abot>
<abot>Dec.</abot>
<abot>Don.</abot>
<abot>Ham.</abot>
<abot>Hamilt.</abot>
<abot>Jon.</abot>
<abot>L.</abot>
<abot>Lam.</abot>
<abot>Lin.</abot>
<abot>Roxb.</abot>
<abot>Roxburghii R. Br.</abot>
<abot>Roxburghii Wall.</abot>
<abot>W. Hunt.</abot>
<abot>W. u. A.</abot>
<abot>Wall.</abot>
<abot>Whigt. Arn.</abot>
```

**Comment by drdhaval2785** [comment]
https://github.com/sanskrit-lexicon/csl-orig/commit/979698daf7c580a9566704da0f320974cced57fb closed this.
